WorkoutTest/ExerciseController.swift

```diff
@@ -72,47 +81,55 @@ class ExerciseController: UIViewController {
   // MARK: - setupView
   
   private func setupView() {
+    view.addSubview(myScrollView)
+    myScrollView.addSubview(myImage)
+    myScrollView.addSubview(nameExercise)
+    myScrollView.addSubview(descriptionLabel)
+    myScrollView.addSubview(backButton)
+    myScrollView.addSubview(nextButton)
```

Comment:
these two buttons should not be placed inside scroll view and should not scroll with content. They should stay on top of the scroll view.